paasta_tools/kubernetes_tools.py

```diff
@@ -515,67 +535,94 @@ def get_desired_instances(self) -> int:
             raise Exception("Number of instances must be 1 or 0 if an EBS volume is defined.")
         return instances
 
-    def format_kubernetes_app(self) -> V1Deployment:
+    def get_volume_claim_templates(self) -> List[V1PersistentVolumeClaim]:
+        return [
+            V1PersistentVolumeClaim(
+                metadata={'name': self.get_persistent_volume_name(volume)},
+                spec=V1PersistentVolumeClaimSpec(
+                    # must be ReadWriteOnce for EBS
+                    access_modes=["ReadWriteOnce"],
+                    storage_class_name=self.get_storage_class_name(),
+                    resources=V1ResourceRequirements(
+                        requests={
+                            'storage': f"{volume['size']}Gi",
```

Comment:
How is resource type `storage` going to work?
Also probably it's better to name it `storage-space` as planned at https://github.com/kubernetes/community/blob/master/contributors/design-proposals/scheduling/resources.md#future-storage-space